src/logion/services/transaction.extractor.ts

```diff
@@ -169,6 +169,56 @@ export class TransactionExtractor {
             delete reservedMap[extrinsic.valueFee.withdrawnFrom];
         }
 
+        if(extrinsic.collectionItemFee && extrinsic.collectionItemFee.beneficiary) {
+            // Requester -> already handled with EXTRINSIC transaction
+            // Beneficiary
+            transactions.push(new Transaction({
+                extrinsicIndex: index,
+                pallet: this.pallet(extrinsic),
+                method: this.methodName(extrinsic),
+                tip: 0n,
+                fees: new Fees({ inclusionFee: 0n }),
+                reserved: 0n,
+                from: extrinsic.collectionItemFee.withdrawnFrom,
+                transferValue: extrinsic.collectionItemFee.received,
+                to: extrinsic.collectionItemFee.beneficiary,
+                type: "COLLECTION_ITEM_FEE",
+                hiddenFrom: "FROM",
+            }));
+        }
+
+        if(extrinsic.tokensRecordFee) {
+            // Requester
+            transactions.push(new Transaction({
+                extrinsicIndex: index,
+                pallet: this.pallet(extrinsic),
+                method: this.methodName(extrinsic),
+                tip: 0n,
+                fees: new Fees({ inclusionFee: 0n, tokensRecordFee: extrinsic.tokensRecordFee.fee }),
+                reserved: 0n,
+                from: extrinsic.tokensRecordFee.withdrawnFrom,
+                transferValue: undefined,
+                to: undefined,
+                type: "TOKENS_RECORD_FEE",
+                hiddenFrom: "TO",
```

Comment:
`to` is undefined here, which totally make sense.
Therefore the `hiddenFrom: "TO"` appears a bit useless, the present transaction will anyway not be listed
(SQL-clause first part `(transaction.to_address = :address AND...` will anyway be false).
But it does not harm of course.
Or did I miss something ?

Reply:
Indeed, just removed the `hiddenFrom`.